### DISCUSSION TITLE
can this be used to simulate a cell membrane, or protein docking?

### DISCUSSION THREAD

**Discussion by dgutson** [NeLy-EPFL/flygym#132] [General]
I'm wondering if the biomechanical modelling implementation could be applied to either cell membranes or protein-protein bound compounds. What authors do think about this?

**Comment by sibocw** [comment]
Depending on what you want to simulate, you can maybe use the [MuJoCo](https://mujoco.org/) simulator underlying NeuroMechFly. Any fly-specific code in this package is unlikely to be directly useful.
